test/fixtures/transform_params.html

```diff
@@ -0,0 +1,3 @@
+<span>{{ content.string | repeat:4 }}</span>
+<span>{{ content.string | replace:'B':'C' }}</span>
```

Comment:
I knew the `{varname|modifier:param1:param2}` syntax form my time with php [smarty](http://www.smarty.net/docs/en/language.modifiers.tpl) and also know it from [angular filters](https://docs.angularjs.org/guide/filter). I like that this syntax binds the transform and its arguments visually:
`{{ myvar | transform:parameter1:parmaeter2:... }}`

----

**Notation with parentheses**

```
{{ myvar | replace('B', 'C') }}
```
Brings association with a function call, which is nice because transforms are functions, but I dislike that `myvar` is *magically* going into the function.

----

**Notation with coma**

```
{{ myvar | replace:'B','C' }}
```
Is also working for me, but form my feeling it is splitting the parameters a bit more apart.

------
**Traditional unix notation** (for going back to the roots)

```
{{myvar | replace "B" "C"}}
_foreach="item : list | max 5"
```
One get lost very easy, not eye friendly at all. 

----

**Finally**: I would stick to the colon syntax as developers may know it from using filters, modifiers, etc. of other libraries.

---

### Some other aspect
However you decide on the notation for the transforms and their parameters, how about saying for the `_foreach` that `list` is a placeholder so it should be defined like
```
_foreach="item:{{list | max: 2}}"
```
This would align with how variables can be used in the destination path in  `_bake`-attribute. Of course we still need to support the basic usage `_foreach="item:list"`.

----
P.S. Sorry for the log post :sweet_potato: